e2e/process-services-cloud/process-custom-filters.e2e.ts

```diff
@@ -124,45 +125,23 @@ describe('Process list cloud', () => {
         it('[C290069] Should display processes ordered by name when Name is selected from sort dropdown', async () => {
             processCloudDemoPage.editProcessFilterCloudComponent().clickCustomiseFilterHeader().setStatusFilterDropDown('RUNNING')
                 .setSortFilterDropDown('Name').setOrderFilterDropDown('ASC');
-            processCloudDemoPage.processListCloudComponent().getAllRowsNameColumn().then(function (list) {
-                const initialList = list.slice(0);
-                list.sort();
-                expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
-            });
+
+            expect(processCloudDemoPage.processListCloudComponent().getDataTable().checkListIsSorted('ASC', 'Name')).toBe(true);
```

Comment:
I suggest adding also a message in case of failure. For boolean expects, in case the expect fails, it will outout ```Expect false to be true``` which doesn't provide much info

Reply:
yep I agree but I guess this is a brother discussion we should have in general on how to write the expectation